### ISSUE TITLE
Drag and Drop add PSD results in Error

### ISSUE THREAD

**Issue by lisafolkerson** [avocode/avocode#1500]
IMPORTANT: Please try to describe how did it happen.

Add project folder, then when drag and drop psd file onto avocode error occurs.

-----
App v. `2.14.4`
Avocode UID: 146408
```
Error: Files over size limit are not supported. (500 MB)
    at Object.normalizeError (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:16855)
    at Object.normalizeResponse (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:16734)
    at t.normalizeResponse (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:1988538)
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:1984599
    at Request._callback (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:1987379)
    at Request.self.callback (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/request/request.js:186:22)
    at emitTwo (events.js:111:20)
    at Request.emit (events.js:191:7)
    at Request.<anonymous> (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/request/request.js:1081:10)
    at emitOne (events.js:96:13)
    at Request.emit (events.js:188:7)
    at IncomingMessage.<anonymous> (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/request/request.js:1001:12)
    at Object.onceWrapper (events.js:290:19)
    at emitNone (events.js:91:20)
    at IncomingMessage.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

**Comment by jakubzitny** [comment]
Hi @lisafolkerson, thanks for reporting the problem. We already know about this and the behaviour will be fixed for `2.14.5`. You'll only see error notification instead of app problem.

The error basically says that we don't support uploading designs that are larger than 500MB. If you want to do that, please download our Photoshop plugin from [avocode.com/download](https://avocode.com/download) or write to our support team at team@avocode.com and they will be happy to upload it for you.

**Comment by jakubzitny** [comment]
We've just released `2.14.5` where we handle this case properly, please update the app and let us know if you have any more problems 😉
